specifyweb/frontend/js_src/lib/components/Merging/Header.tsx

```diff
@@ -132,16 +132,18 @@ function SummaryLines({
 export function MergeRow({
   header,
   children,
+  className = '',
```

Comment:
I think we do that a lot. Search `className?` in the codebase

Reply:
Well, I guess we _do_ do this a lot :laughing: 
It occurs as an optional prop in 13 places and as a required prop (in functions which are exported) in 2 places. 

There are some benefits to this approach: it allows the consumer of the function to customize the behavior of the component it is rendering 
Although, I can imagine this can _also_ be a hindrance if used to _either_ an extensive or limited degree:
From a UX consistency point of view and code maintenance view, respectively.  

I would be more inclined to accept this if there were more of a use for the className prop outside of the one place, but this _also_ seems to be a common occurrence in the cases where className is an optional prop... 

Anyways, this change is fine!
This comment is more subjective than objective, and should not hold up merging this when necessary

Reply:
What would be a better way?